jwst/associations/lib/rules_level3_base.py

```diff
@@ -802,13 +802,12 @@ def __init__(self, *args, **kwargs):
             name='acq_constraint',
             work_over=ProcessList.EXISTING
         )
-
         # Put all constraints together.
         self.constraints = Constraint(
             [
                 Constraint_Base(),
                 DMSAttrConstraint(
-                    sources=['is_imprt', 'bkgdtarg'],
+                    sources=['is_imprt'],
```

Comment:
The effect of this change can be that backgrounds will end up as backgrounds for themselves. Suggested approach is to create another rule that processes such exposures separately.